docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -106,3 +108,45 @@ Used to lookup for suggestions for the new name by returning 3 unique usernames.
 ```
 
 * `suggestions` - suggested profile names.
+
+### Register or update name
+
+Used to register account name.
+
+`POST /v1/profile/account/{name}`
+
+* `name` - is the name to register e.g. `coolname.connect.id`
+
+#### Request body:
+
+The POST request body should be in JSON format with the folowing structure:
+
+```typescript
+{
+    "message": string,
+    "signature": string,
+    "address":string,
+}
+```
+
+* `message` - JSON serialized string that should contain the following fields:
+
+    * `name` - is the name to register or update.
+        * Alphabetic characters (A-Z a-z), numeric characters (0-9), the minus sign (-) and the period (.)
+        * Max 255 characters length.
+    * `address` - address that bind to the account name.
+        * Max 100 characters length.
+    * `timestamp` - current unixtime timestamp. The signature is valid for 30 seconds.
```

Comment:
Adding the 30 seconds validity of the signature, to limit replay attack.

Reply:
Might be a bit short to accommodate clock drift, I'd do 5 minutes to be safe.

Reply:
The reference for the 30 seconds for my decision is the [TOTP](https://ru.wikipedia.org/wiki/Time-based_One-time_Password_Algorithm) it's time-based and it has a 30 seconds gap for the sync.